src/matter/cluster/PressureMeasurementCluster.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8 } from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for Pressure Measurement.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.4
+ */
+export const PressureMeasurementCluster = Cluster({
+    id: 0x0403,
+    name: "PressureMeasurement",
+    revision: 3,
+    features: {
+        /** The cluster is capable of extended range and resolution */
+        extended: BitFlag(0) 
+    },
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.4.5 */
+    attributes: {
+       /** Represents the pressure in kPa as follows: MeasuredValue = 10 x Pressure [kPa] */
+       measuredValue: Attribute(0x0, TlvNullable(TlvInt16)),
+       
+       /** Indicates the minimum value of MeasuredValue that can be measured. */
+       minMeasuredValue: Attribute(0x1, TlvNullable(TlvInt16)),
```

Comment:
I would propose to add the min: -32767 here because int16 min is normally -32768 ...